pandas/tests/io/excel/test_readers.py

```diff
@@ -148,6 +149,53 @@ def parser(self, *args, **kwargs):
             expected = expected_defaults[read_ext[1:]]
         assert result == expected
 
+    def test_engine_kwargs_valid(self, read_ext, engine):
```

Comment:
Does this test cover something the other test doesn't?

Reply:
@rhshadrach 
The test_engine_kwargs_valid function tests valid arguments being passed to the load_workbook method for each engine.

The invalid method tests args that are not supported the various engines

Reply:
> The test_engine_kwargs_valid function tests valid arguments being passed to the load_workbook method for each engine.

It doesn't do this. It only tests that the call returns a non-empty frame. This test will still pass even if we didn't pass the arguments to the engine.